ojgl/src/render/Buffer.cpp

```diff
@@ -158,27 +158,28 @@ ojstd::string Buffer::name() const
     return _name;
 }
 
-#define GL_CHECK_ERROR(functionCall) functionCall; \
-    { \
-        GLenum error = glGetError(); \
-        if (error != GL_NO_ERROR) { \
-            _ASSERTE(false); \
+#define GL_CHECK_ERROR(functionCall)          \
+    functionCall;                             \
+    {                                         \
+        GLenum error = glGetError();          \
+        if (error != GL_NO_ERROR) {           \
+            _ASSERTE(false);                  \
             LOG_ERROR("GL Error: " << error); \
-        } \
+        }                                     \
     }
 
-#define GL_CHECK_LOG(functionCall, id, logFunction, errorStr) functionCall; \
-    functionCall; \
-    if (param == GL_FALSE) {\
-        LOG_ERROR(errorStr); \
-        int len; \
-        constexpr int logSize = 200; \
-        char log[logSize]; \
-        logFunction(id, logSize, &len, log); \
+#define GL_CHECK_LOG(functionCall, id, logFunction, errorStr)                       \
+    functionCall;                                                                   \
+    functionCall;                                                                   \
```

Comment:
Oops, should only be a single functionCall, I missed that when I initially made it